### ISSUE TITLE
Wildfire Aware - Link to this application? 

### ISSUE THREAD

**Issue by pjdohertygis** [pjdohertygis/FireMappers#43]
Since Wildfire Aware is such an awesome app, although different audience from FireMappers, should we link out to Wildfire Aware with a button link? https://livingatlas.arcgis.com/wildfireware/

Also @avezina notice their URL parameters and relationship with the "list widget". Looks like custom stuff? 
https://livingatlas.arcgis.com/wildfireware/#@=-114.980,43.811,12,7d82a608-618c-4892-a1d7-514e673379fa

![image](https://user-images.githubusercontent.com/4256979/190196559-c19c7f99-096b-4aac-9140-dc34d76322fd.png)

**Comment by pjdohertygis** [comment]
It actually looks like a "hack". The URL parameter has a lat/long (which can be feature drive by our layers) and then an extent based filter for the "list". This may be a viable approach for ExB???

**Comment by avezina** [comment]
I don't believe this is Experience Builder, looks like a custom JavaScript app. URL params would work however they set it up.
